### DISCUSSION TITLE
Git error and sync error

### DISCUSSION THREAD

**Discussion by NRanjan-17** [ROM-builders/temporary#8556] [General]
Sir I am getting Git error and sync error in my builds.
Please fix it.
https://cirrus-ci.com/build/6243109678874624
sir this is only one i am getting this in majority if my sync is successful than also git error occurs for a single time

**Comment by Apon77** [comment]
> Sir I am getting Git error and sync error in my builds. Please fix it. https://cirrus-ci.com/build/6243109678874624 sir this is only one i am getting this in majority if my sync is successful than also git error occurs for a single time

what can i do regarding your last line?

**Comment by NRanjan-17** [comment]
Please check if git error fix is working or not

**Comment by rk134** [comment]
Hi!

The sync error is usually due to a branch (revision) error, and this can be seen in this line

`error: Cannot checkout vendor_xiaomi_mido: ManifestInvalidRevisionError: revision twelve in vendor_xiaomi_mido not found`


The error for this fix is to correct your repository name, and double check the names of your branch that are in local_manifest.

example:
repository name: vendor_xiaomi_mido
branch name: twelve-test
remote="mido" 

```
<remote name="mido"
              fetch="https://github.com/example/vendor_xiaomi_mido" />

<project path "vendor/xiaomi/mido" name="vendor_xiaomi_mido" remote="mido" revision="twelve-test" />
```

**Hope this answer helps you! If it does, kindly mark the discussion as answered, to close the thread.**
